web/src/app/profile/approved-researcher/page.tsx

```diff
@@ -9,9 +9,9 @@ export default function ApprovedResearcherPage() {
   return (
     <div className="profile-page">
       <h1>Become an Approved Researcher</h1>
-      <p>To gain access to advanced services, please complete the following agreement.</p>
```

Comment:
Is the `className="profile-page"` still appropriate here? The same className is used in the actual `ProfilePage` component.

Reply:
I could fix this in the renaming PR if preferred

Reply:
which doesn't exist – I'll rm!